sympy/core/evalf.py

```diff
@@ -1186,7 +1186,7 @@ def evalf_sum(expr, prec, options):
     if len(limits) != 1 or len(limits[0]) != 3:
         raise NotImplementedError
     if func is S.Zero:
-        return mpf(0), None, None, None
+        return None, None, None, None
```

Comment:
I would set the third item equal to `prec`. That is how it is in `evalf.evalf(S(0))`. (Though I don't know it the value is actually used currently.)

Reply:
Yes,already done